github-runners/k8s/templates/base/acme-certificate.yaml

```diff
@@ -0,0 +1,67 @@
+apiVersion: cert-manager.io/v1
+kind: ClusterIssuer
+metadata:
+  name: letsencrypt-production
+  namespace: explorer-network
```

Comment:
Strange namespace `explorer-network`. Also do we need most of this commented-out file?